### ISSUE TITLE
Question about the attention

### ISSUE THREAD

**Issue by Han8931** [bentrevett/pytorch-seq2seq#6]
Hello

I have a question about your attention code.

In the decoder code, you concatenate the embedded and the context.
Is this the way how the original paper did?
and if not, could you explain why you concatenated them?

        rnn_input = torch.cat((embedded, context), dim=2)
        
        #rnn_input = [1, batch size, (enc hid dim * 2) + emb dim]
            
        output, hidden = self.rnn(rnn_input, hidden.unsqueeze(0))

**Comment by Han8931** [comment]
Thanks a lot! :)
